tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/UserSteps.cs

```diff
@@ -101,51 +101,78 @@ public async Task ThenTheUsersListIsReturnedWithValues(Table table)
 
         private static string GenerateHash(string password)
         {
-            using (var rng = RandomNumberGenerator.Create())
+            const int IdentityVersion = 1; // 1 = Identity V3
+            const int IterationCount = 10000;
+            const int PasswordHashLength = 32;
+            const KeyDerivationPrf HashAlgorithm = KeyDerivationPrf.HMACSHA256;
+            const int SaltLength = 16;
+
+            using var rng = RandomNumberGenerator.Create();
+            var salt = new byte[SaltLength];
+            rng.GetBytes(salt); //The GetBytes method fills the salt array with random data
```

Comment:
I don't think the comment is adding any value – we can get the same info from the XML docs/IntelliSense.